src/mlpack/methods/dbscan/dbscan_main.cpp

```diff
@@ -123,13 +123,20 @@ static void mlpackMain()
       "hilbert-r", "r-plus", "r-plus-plus", "ball" }, true,
       "unknown tree type");
 
+  // Value of epsilon should be positive.
+  RequireParamValue<double>("epsilon", [](double x) { return x > 0; },
+      true, "invalid value of epsilon specified");
+
+  // Value of min_size should be positive.
+  RequireParamValue<int>("min_size", [](int y) { return y > 0; },
+      true, "invalid value of min_size specified");
+
   // Fire off naive search if needed.
   if (CLI::HasParam("naive"))
   {
     RangeSearch<> rs(true);
     RunDBSCAN(rs);
-  }
-
+  } else {
```

Comment:
I know the style checker is not currently working, but can you reformat this line as

```
}
else
{
```

and update the spacing for the code now inside the else block? Thanks!